flink-connector-oceanbase/src/test/java/com/oceanbase/connector/flink/OceanBaseOracleConnectorITCase.java

```diff
@@ -0,0 +1,537 @@
+/*
+ * Copyright 2024 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.connector.flink;
+
+import com.oceanbase.connector.flink.connection.OceanBaseConnectionProvider;
+import com.oceanbase.connector.flink.dialect.OceanBaseDialect;
+import com.oceanbase.connector.flink.dialect.OceanBaseOracleDialect;
+import com.oceanbase.connector.flink.sink.OceanBaseRecordFlusher;
+import com.oceanbase.connector.flink.sink.OceanBaseSink;
+import com.oceanbase.connector.flink.table.DataChangeRecord;
+import com.oceanbase.connector.flink.table.OceanBaseRowDataSerializationSchema;
+import com.oceanbase.connector.flink.table.OceanBaseTestData;
+import com.oceanbase.connector.flink.table.OceanBaseTestDataSerializationSchema;
+import com.oceanbase.connector.flink.table.SchemaChangeRecord;
+import com.oceanbase.connector.flink.table.TableId;
+import com.oceanbase.connector.flink.table.TableInfo;
+import com.oceanbase.connector.flink.utils.OceanBaseJdbcUtils;
+
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+import org.apache.flink.table.catalog.Column;
+import org.apache.flink.table.catalog.ResolvedSchema;
+import org.apache.flink.table.catalog.UniqueConstraint;
+import org.apache.flink.table.data.DecimalData;
+import org.apache.flink.table.data.GenericRowData;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.data.StringData;
+import org.apache.flink.types.RowKind;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@Ignore
+public class OceanBaseOracleConnectorITCase extends OceanBaseOracleTestBase {
```

Comment:
There should be a test case for case-insensitive option, and I think other cases are not necessary for now.

Reply:
I kept two basic test cases and they passed in the dev env。